openassessment/assessment/test/test_staff.py

```diff
@@ -0,0 +1,226 @@
+                },
+                {
+                    "name": "reluctant",
+                    "points": "3",
+                    "explanation": "May argue, but will give it up voluntarily."
+                },
+                {
+                    "name": "eager",
+                    "points": "10",
+                    "explanation": "Happy to give it up."
+                }
+            ]
+        },
+        {
+            "name": "singing",
+            "prompt": "Did the writer break into tedious elvish lyrics?",
+            "options": [
+                {"name": "no", "points": "2", "explanation": ""},
+                {"name": "yes", "points": "0", "explanation": ""}
+            ]
+        },
+    ]
+}
+
+# Answers are against RUBRIC_DICT -- this is worth 6 points
+ASSESSMENT_DICT = {
+    'overall_feedback': u"这是中国",
+    'criterion_feedback': {
+        "giveup": u"𝓨𝓸𝓾 𝓼𝓱𝓸𝓾𝓵𝓭𝓷'𝓽 𝓰𝓲𝓿𝓮 𝓾𝓹!"
+    },
+    'options_selected': {
+        "secret": "yes",
+        u"ⓢⓐⓕⓔ": "no",
+        "giveup": "reluctant",
+        "singing": "no",
+    },
+}
+
+# Answers are against RUBRIC_DICT -- this is worth 0 points
+ASSESSMENT_DICT_FAIL = {
+    'overall_feedback': u"fail",
+    'criterion_feedback': {},
+    'options_selected': {
+        "secret": "no",
+        u"ⓢⓐⓕⓔ": "no",
+        "giveup": "unwilling",
+        "singing": "yes",
+    }
+}
+
+# Answers are against RUBRIC_DICT -- this is worth 14 points
+ASSESSMENT_DICT_PASS = {
+    'overall_feedback': u"这是中国",
+    'criterion_feedback': {},
+    'options_selected': {
+        "secret": "yes",
+        u"ⓢⓐⓕⓔ": "yes",
+        "giveup": "eager",
+        "singing": "no",
+    }
+}
+
+# Answers are against RUBRIC_DICT -- this is worth 14 points
+# Feedback text is one character over the limit.
+LONG_FEEDBACK_TEXT = u"是" * Assessment.MAX_FEEDBACK_SIZE + "."
+ASSESSMENT_DICT_HUGE = {
+    'overall_feedback': LONG_FEEDBACK_TEXT,
+    'criterion_feedback': {
+        "secret": LONG_FEEDBACK_TEXT,
+        u"ⓢⓐⓕⓔ": LONG_FEEDBACK_TEXT,
+        "giveup": LONG_FEEDBACK_TEXT,
+        "singing": LONG_FEEDBACK_TEXT,
+    },
+    'options_selected': {
+        "secret": "yes",
+        u"ⓢⓐⓕⓔ": "yes",
+        "giveup": "eager",
+        "singing": "no",
+    },
+}
+
+
+
+@ddt
+class TestStaffOverwrite(CacheResetTest):
+    """
+    Tests for staff assessments made as overrides, when none is required to exist.
+    """
+
+    STEPS = ['self']
+    STEP_REQUIREMENTS = {}
+
+    def test_create_assessment_points(self):
+        tim_sub, tim = self.create_student_and_submission("Tim", "Tim's answer")
+
+        assessment = staff_api.create_assessment(
+            tim_sub["uuid"],
+            "Dumbledore",
+            ASSESSMENT_DICT['options_selected'], dict(), "",
+            RUBRIC_DICT,
+        )
+        self.assertEqual(assessment["points_earned"], 6)
+        self.assertEqual(assessment["points_possible"], 14)
+
+        #ensure submission is marked as finished now
```

Comment:
Yeah, I forgot to write that assertion, good catch.

I've included it in the test PR I'm working on here: https://github.com/edx/edx-ora2/pull/768